about.html

```diff
@@ -0,0 +1,113 @@
+    <link rel="stylesheet" href="styles.css">
+    <title>Green Earth Summit</title>
+</head>
+<body>
+    <nav>
+        <div class="upper-nav">
+            <ul class="top">
+                <li class="fb"><img  id="fb" src="assets/fb (1).png" alt="facebook icon"/></li>
+                <li class="twitter"><img id="twitter" src="assets/twitter.png" alt="twitter icon"></li>
+                <li class="pa"><p>English</p></li>
+                <li class="pa"><p>MyPage</p></li>
+                <li class="pa"><p>Logout</p></li>
+            </ul>
+        </div>
+        <div class="hamburger">
+            <ul>
+            <li class="bar"></li>
+            <li class="bar"></li>
+            <li class="bar"></li>
+        </ul>
+        </div>
+        <div  class="flex" >
+            <a  class="flex logo" href="index.html"><img  id="logo"  src="assets/Earth.png"/></a>
+            <ul class="flex nav-menu">
+              <li class="nav-item">
+              <a class="nav-link" id="nav1"  href="index.html">Home</a>
+            </li>
+            <li class="nav-item">
+              <a class="nav-link" href="#">Program</a>
+            </li>
+            <li class="nav-item">
+                <a class="nav-link" href="#">Join</a>
+              </li>
+              <li class="nav-item">
+                <a class="nav-link" href="#">Sponsor</a>
+              </li>
+            <li class="nav-item">
+              <a class="nav-link"  href="#">News</a>
+            </li>         
+            <li class="nav-item GE">
+                <a class="nav-link" id="ge" href="#">Campaign</a>
+              </li>    
+          </ul>
+          </div>
+    </nav>
+   <!--Header-->
+   <header>
+    <h1 class="heading1about">"Hello!Sharing world"</h1>
+    <h2 class="heading2">Green Earth Global Summit 2023</h2>
+    <p  class="paragabout">Human existence is quite impossible without the presence of a healthy ecosystem. Our environment comprises all living and non-living components and their interactions within a natural habitat. Environmental conservation has become one of the core issues that need to be addressed to battle climate change and global warming.Green Earth Global aims at conservation of natural resources. </p>
+      </header>
+
+<!-- more info -->
+  <section class="info-sec">
+    <p class="info-p">please contact us for further questions<br> regarding our annual summit<br>
+        <a class="info" href="#">info@GreenEarth.org</a>
+    </p>
+  </section>
+
+<!-- past summit -->
+<section class="past-summit">
+    <h3 class="summit">Green Earth Global Summit 2020</h3>
+    <hr class="hrabout">
+    <p class="majisafiP">The logo of Green Earth organisation was changed as of 21st July, 2020. We thank everyone who took part in voting for the logo below.</p>
+    <img class="majisafi" src="assets/Earth.png" alt="logo"/>
+</section>
+
+<!-- Past pictures -->
+<section class="pastphotos">
+    <h4 class="pastp">See the past G.E Global summit </h4>
```

Comment:
- [ ] Please note the following design discrepancies and align them in your implementation

**Design**
![midl1](https://github.com/fatmahussein/Green-Earth-Summit/assets/99656687/6fe3cfa8-656c-45c0-ba4a-9669d271850e)

**Impelentation**
![midl2](https://github.com/fatmahussein/Green-Earth-Summit/assets/99656687/032a89a4-a6b3-48c2-a6a8-d55321ea00e4)

**Please take the following action**
- [ ] Add spacing between the title & the top section 😀 
- [ ] Color the divers red 🦆 
- [ ] Note the dividing line and add it 👍🏾